src/PhpSpreadsheet/Style/Alignment.php

```diff
@@ -475,6 +512,7 @@ public function getHashCode(): string
 
         return md5(
             $this->horizontal
+            . $this->justifyLastLine
```

Comment:
Another case where the 3-pronged value will cause problems. Probably change this to:
```php
. (($this->justifyLastLine === null) ? 'null' : ($this->justifyLastLine ? 't' : 'f'))
```